astropy/coordinates/tests/test_transformations.py

```diff
@@ -645,39 +651,42 @@ class H3(HCRS):
     assert transform3.finite_difference_dt == old_dt * 3
 
 
-# fmt: off
```

Comment:
Weird that my comment was lost, but this is the one that I felt was better as it was (though I would add a newline after `parametrize(` or move the opening bracket of the list to the first line.

It might also be possible to get things more on one line by using `ROT_30` etc instead of `ROTATION_30`.

Reply:
Is it better now?

Reply:
Yes, quite a bit -- and good enough for me at least!